src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
+  --Message-background: var(--color-gray);
+  --Message-offline-background: var(--color-orange);
+  --Message-offline-color: var(--color-white);
+  --Message-close-size: 1.4em;
+}
+
+.Message {
+  position: relative;
+  padding: var(--Message-padding);
+  background: var(--Message-background);
+  text-align: center;
+}
+
+.Message-close {
+  position: absolute;
+  display: block;
+  top: var(--space-xs);
+  right: var(--space-xs);
+}
+
+.Message-close svg {
+  transform: rotate(45deg); /* turns a plus sign into an X */
```

Comment:
It would be cleaner to create a new `x.svg` icon with the transform attribute I mentioned in Slack.

It's source should be this:

```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24">
  <g transform="rotate(45 12 12)" fill="none" stroke="currentColor" stroke-width="4" stroke-linecap="round">
    <line x1="12" y1="3" x2="12" y2="21"/>
    <line x1="3" y1="12" x2="21" y2="12"/>
  </g>
</svg>
```

Then you can just update the template.